engines/bops_core/app/controllers/concerns/bops_core/auditable_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module BopsCore
+  module AuditableController
+    extend ActiveSupport::Concern
+
+    AUDITABLE_ACTIONS = {
+      "create" => "created",
+      "update" => "updated",
+      "destroy" => "destroyed"
+    }.freeze
+
+    included do
+      class_attribute :audit_payload, instance_writer: false, default: -> { {} }
```

Comment:
Looking to extract this to a shared concern and add support for calling `audit` anywhere